front-end/src/components/main-page/MainPage.jsx

```diff
@@ -17,16 +17,27 @@ class MainPage extends Component {
     this.handleChange = this.handleChange.bind(this);
     this.state = {
       showTranscriptUploadModal: false,
+      useMerger: false,
     };
   }
 
   toggleTranscriptUploadModal = () => {
     this.setState({showTranscriptUploadModal: !this.state.showTranscriptUploadModal});
   }
 
+  toggleMerge = () => {
+    this.setState({useMerger: !this.state.useMerger});
+  }
+
   handleChange = (event) => {
     // Update the Project ID of the analyzed transcript to display in redux based on input field
-    store.dispatch(setProjectIdToBeDisplayed(event.target.value));
+    const enteredId = event.target.value
+    if(enteredId){
```

Comment:
Maybe try to merge this into one line IMO. The variable extract adds a little clutter to me. Maybe get other's opinions